pkg/observability/metrics/prometheus/handler.go

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Instead of creating new files in the package, it's better to move the existing files here. That way the change history of the files is preserved. (And it also makes this PR much smaller :) )

Reply:
I agree (forgot to do the cleanup)